extension.js

```diff
@@ -333,6 +337,10 @@ function unmaximize_window(win) {
     if (!win.maximized_vertically)
         return;
 
+    const maximized = settings.get_boolean('window-maximize');
+    if (maximized)
```

Comment:
Please simply use `settings.get_boolean('window-maximize')` if its value is used only once (and if the expression isn't too long). When you introduce a variable like this, it makes me expect that it will be used later